### ISSUE TITLE
Invalid config.js files are ignored as if they don't exist (with no warning)

### ISSUE THREAD

**Issue by Rahe** [renovatebot/renovate#4894]
Hello ,

### Which Renovate are you using? CLI, App, or Pro
CLI

### Which platform are you using? GitHub, GitLab, Bitbucket Azure DevOps
Bitbucket cloud

### Have you checked the logs? Don't forget to include them if relevant
Here is my config.js file :
```
module.exports = {
  platform: 'bitbucket',
  username: 'username'
  password:'password'
  logFileLevel: 'warn',
  logLevel: 'info',
  logFile: '/Users/nicolasjuen/renovate.log',
  onboarding: true,
  gitAuthor: "Renovate Bot <bot@renovateapp.com>"
  onboardingConfig: {
    extends: ['config:base'],
  },
  repositories: [ 'company/rep-name' ],
};
```

When I run on the folder containing the config.js file :
`renovate`
I have
`FATAL: You must configure a GitHub personal access token`

If I run 
`RENOVATE_CONFIG_FILE=/Users/nicolasjuen/.renovate/config.js renovate`
I have
`FATAL: You must configure a GitHub personal access token`

The only way I have this working is to specify 
--password
--username
--platform
and the repo at the end...

Why is my file ignored ?

**Comment by rarkins** [comment]
I've transferred this into the main repo and reopened it so that it's in the issue queue as something to do.

**Comment by souravdasslg** [comment]
Checking if the error type is "SyntaxError" will do the work.
Do we have anything else here to handle?

**Comment by rarkins** [comment]
Yes it’s possible that the errors are different but otherwise doesn’t check if it exists and then require it so you know which step failed

**Comment by rarkins** [comment]
If the file doesn’t exist, we do nothing (like today). If it exists but we can’t parse it, we should error and skip the repo

**Comment by rarkins** [comment]
Fixed in #5057
